plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -163,7 +163,7 @@ def start(self) -> None:
             env=os.environ.copy(),
         )
         empty_outputs = 0
-        for stdout_line in iter(self.process.stdout.readline, ""):
+        for stdout_line in iter(cast(IO[bytes], self.process.stdout).readline, ""):
```

Comment:
Cast is not always the right solution. Here the stdout could be None. So we should have an explicit check for this and then raise exception if so imo